packages/polymath-auth/src/networks.js

```diff
@@ -27,11 +26,10 @@ export default (id: string = 'local'): Network =>
     },
     [NETWORK_KOVAN]: {
       name: 'Kovan Testnet',
-      url:
-        'wss://heartily-internal-escargot.quiknode.io/46df7525-5518-428e-b30b-9dea09480213/4oM662IRx_2tZJq3ht4wdQ==/',
+      url: process.env.REACT_APP_NETWORK_KOVAN_WS,
     },
     local: {
       name: 'Localhost',
-      url: 'ws://localhost:8545',
+      url: process.env.REACT_APP_NETWORK_LOCAL_WS,
```

Comment:
For this one, I'd suggest to do something like `process.env.REACT_APP_NETWORK_LOCAL_WS || 'ws://localhost:8545'`